src/components/ResponsiveNEWMLogo.tsx

```diff
@@ -0,0 +1,18 @@
+import { FunctionComponent } from "react";
```

Comment:
I created this new logo component bc i couldnt find one that would resize by default. Maybe this was an overkill and would like to know if there are other ways to get the responsive part included